src/components/backend-ai-storage-list.ts

```diff
@@ -401,6 +401,9 @@ export default class BackendAiStorageList extends BackendAIPage {
           --mdc-typography-font-family: var(--general-font-family);
           --mdc-typography-label-font-size: 12px;
           --mdc-theme-primary: var(--general-textfield-selected-color);
+          /* Need to be set when fixedMenuPosition attribute is enabled */
+          --mdc-menu-max-width: 320px;
+          --mdc-menu-min-width: 320px;
         }
```

Comment:
Shouldn't be more appropriate to locate this style definition in "`mwc-select.fixed-position`" ?